articles/understanding-systemd/index.md

```diff
@@ -0,0 +1,248 @@
+## Introduction
+From the official website, systemd (intentionally spelt with a small 's') is described as a suite of tools that forms the basic building block for a Linux system.
+systemd is primarily responsible for handling the boot process for Linux systems. As an init implementation, it has a PID of 1 just like other init systems e.g System V, Upstart.
+
+### What's init?
+Before we delve further, lets talk more on what exactly init is. 
+An init program is usually the first to start and the last to stop. It initializes and terminates programs and services crucial to the operation of Linux systems. It's actually a daemon process that runs during the entire life time of all other programs and services in the system. It's acts as the parent process such that when child processes are orphaned or become zombies, the init process takes the responsibility of ensuring that they're properly terminated. 
+Now, what happens when the init process doesn't start first (the kernel doesn't boot the init process)? A kernel panic occurs because there isn't an init process to startup things and make way for essential services.
+
+### Implementations of init
+There are quite a number of init implementations. They usually differ by the scope of their responsibility. systemd for instance is capable of a wide arrays of things that some might argue on the premise of being outside the scope of what an init system should do and the Unix golden rule of doing one thing. The responsibilities common amongst init includes: starting daemons and services, mounting filesystems, configuring network interfaces etc.
```

Comment:
Grammar and long sentence which is quite hard to read.Fix this article on Grammarly.

Reply:
Alright, I've done that